demos/theme/index.html

```diff
@@ -457,6 +457,262 @@ <h2 class="mdc-typography--headline demo-component-section__heading">
+          <span class="mdc-select__selected-text">Pick a food group</span>
+          <div class="mdc-simple-menu mdc-select__menu">
+            <ul class="mdc-list mdc-simple-menu__items">
+              <li class="mdc-list-item" role="option" id="grains" aria-disabled="true">
+                Pick a food group
+              </li>
+              <li class="mdc-list-item" role="option" id="grains" tabindex="0">
+                Bread, Cereal, Rice, and Pasta
+              </li>
+              <li class="mdc-list-item" role="option" id="vegetables" aria-disabled="true" tabindex="0">
+                Vegetables
+              </li>
+              <li class="mdc-list-item" role="option" id="fruit" tabindex="0">
+                Fruit
+              </li>
+              <li class="mdc-list-item" role="option" id="dairy" tabindex="0">
+                Milk, Yogurt, and Cheese
+              </li>
+              <li class="mdc-list-item" role="option" id="meat" tabindex="0">
+                Meat, Poultry, Fish, Dry Beans, Eggs, and Nuts
+              </li>
+              <li class="mdc-list-item" role="option" id="fats" tabindex="0">
+                Fats, Oils, and Sweets
+              </li>
+            </ul>
+          </div>
+        </div>
+      </section>
+
+      <section class="demo-component-section">
+        <i id="slider" class="demo-anchor-with-toolbar-offset"></i>
+        <h2 class="mdc-typography--headline demo-component-section__heading">
+          Slider
+          <a href="#slider" class="demo-component-section__permalink" title="Permalink to the theme demo for the slider component">#</a>
+        </h2>
+
+        <div class="mdc-slider mdc-slider--discrete mdc-slider--display-markers" tabindex="0" role="slider"
+             aria-valuemin="0" aria-valuemax="100" aria-valuenow="30"
+             aria-labelledby="example-ds-marker-label">
+          <div class="mdc-slider__track-container">
+            <div class="mdc-slider__track"></div>
+            <div class="mdc-slider__track-marker-container"></div>
+          </div>
+          <div class="mdc-slider__thumb-container">
+            <div class="mdc-slider__pin">
+              <span class="mdc-slider__pin-value-marker">30</span>
+            </div>
+            <svg class="mdc-slider__thumb" width="21" height="21">
+              <circle cx="10.5" cy="10.5" r="7.875"></circle>
+            </svg>
+            <div class="mdc-slider__focus-ring"></div>
+          </div>
+        </div>
+      </section>
+
+      <section class="demo-component-section">
+        <i id="switch" class="demo-anchor-with-toolbar-offset"></i>
+        <h2 class="mdc-typography--headline demo-component-section__heading">
+          Switch
+          <a href="#switch" class="demo-component-section__permalink" title="Permalink to the theme demo for the switch component">#</a>
+        </h2>
+
+        <fieldset class="demo-switch-wrap">
```

Comment:
Nit: "wrapper" in place of "wrap"? "wrap" can have other connotations...

Reply:
Done.